comms/src/outbound_message_service/outbound_message_pool.rs

```diff
@@ -58,6 +62,7 @@ impl Default for OutboundMessagePoolConfig {
             max_num_of_retries: 10,
             retry_wait_time: Duration::seconds(3600),
             worker_timeout_in_ms: 100,
+            control_timeout_in_ms: 5,
```

Comment:
Yup, it just would look cleaner to use Duration throughout.

Reply:
I agree, I will make the change